chia/rpc/data_layer_rpc_util.py

```diff
@@ -45,11 +45,16 @@ def decorator(route: UnboundMarshalledRoute) -> UnboundRoute:
         request_class: Type[MarshallableProtocol] = hints["request"]
 
         async def wrapper(self: object, request: Dict[str, object]) -> Dict[str, object]:
+            # import json
```

Comment:
do we want to keep these comments? (and `print(f"\n ==== {name} response.json\n{json.dumps(marshalled_response, indent=4)}"`) below)

Reply:
It's naughty, you can make me change this.  I used them for dumping the reference request and response values.  https://gist.github.com/altendky/4b57b25d711f4b33035857ede6388a5b  I think it should become a formal feature in general so we can have tests that validate the request and response and generate easily usable reference.

But...  commented out code is...  dirty.  If you want it removed, I will.